src/PresentationalComponents/Cards/Pathways.js

```diff
@@ -196,34 +198,35 @@ const Resolution = (props) => {
       <div className="flex-coloumn">
         <CardTitle>{intl.formatMessage(messages.resolution)}</CardTitle>
 
-        <div className="flex-row">
-          <div>
+        <div className="flex-row ">
+          <div className="halfWidth">
```

Comment:
Wondering if we can achieve half width layout with a Grid system instead of plain css? I have not tried myself, if this does not make sense please ignore this :)

Reply:
Grid comes with some default css, thats needs to be overwritten if I bring it in. Eventually I'd like to change most of this with <Flex> and <FlexItem> Elements, via a function. I had opted for this for now because using the Flex elements makes the file look kind of crazy, and at least right now I think this version is much more readable